applications/RomApplication/custom_utilities/rom_auxiliary_utilities.cpp

```diff
@@ -469,4 +469,36 @@ void RomAuxiliaryUtilities::GetPhiElemental(
         }
     }
 
+void RomAuxiliaryUtilities::GetPsiElemental(
+        Matrix &rPsiElemental,
+        const Element::DofsVectorType& rDofs,
+        const Element::GeometryType& rGeom,
+        const std::unordered_map<Kratos::VariableData::KeyType, Matrix::size_type>& rVarToRowMapping)
```

Comment:
Our convention uses 4 spaces as indentation.

Reply:
I don't see where it is not well indented.